license: fix copyright owner

jeppenodgaard opened the pull request:
Change license owner to Prevas due to initially wrong owner due to company
mix-up during co-development.

Commit:
license: fix copyright owner

Change license owner to Prevas due to initially wrong owner due to company
mix-up during co-development.

Signed-off-by: Jeppe Odgaard <jeppe.odgaard@prevas.dk>

jeppenodgaard commented:
> @jeppenodgaard can you please add more context in the commit message? Anyone stumbling upon this commit will likely be confused as changing copyright header with no explanation whatsoever is a bit unusual :)

I added a very brief explanation

microbuilder commented:
> I added a very brief explanation

I'm not sure this gives much context to decide if this should be merged or not:

> Change license owner to Prevas due to initially wrong owner.

Does the original legal entity no longer exist? Are you the new IP owner?

jeppenodgaard commented:
> > I added a very brief explanation
> 
> I'm not sure this gives much context to decide if this should be merged or not:
> 
> > Change license owner to Prevas due to initially wrong owner.
> 
> Does the original legal entity no longer exist? Are you the new IP owner?

I updated the commit title and body. 

I hope that add enough context - otherwise let me know.

dkalowsk reviewed (approved):
Still a little confused on what the mix up was, but I guess that isn't something to include here.

MaureenHelm reviewed (commented):
> > I added a very brief explanation
> 
> I'm not sure this gives much context to decide if this should be merged or not:
> 
> > Change license owner to Prevas due to initially wrong owner.
> 
> Does the original legal entity no longer exist? Are you the new IP owner?

I'm not sure about this either. Can someone from Vitrolife confirm that Prevas should be the copyright holder?

jeppenodgaard commented:
> I'm not sure about this either. Can someone from Vitrolife confirm that Prevas should be the copyright holder?

Yes. How should this be communicated?

jkrukow commented:
> > > I'm not sure about this either. Can someone from Vitrolife confirm that Prevas should be the copyright holder?
> > 
> > 
> > Yes. How should this be communicated?
> 
> A comment on this PR

Change of license owner to Prevas is in agreement with Vitrolife. (I work as a software developer at Vitrolife.)
